### ISSUE TITLE
Domain chimerajs.com deactivated? Postinstall script can't download qt binaries

### ISSUE THREAD

**Issue by kloshih** [deanmao/node-chimera#47]
Master's scripts/download.js refers to s3.chimerajs.com to download qt binaries and it looks like the domain chimerajs.com has been Enom'd.

**Comment by basicallydan** [comment]
Yeah I'm having the same issue. This is sad :( I was really hoping for a decent headless webkit for node.
